runtime/bigarray.c

```diff
@@ -339,6 +342,96 @@ CAMLexport intnat caml_ba_hash(value v)
   return h;
 }
 
+CAMLexport void caml_ba_hash_ext(struct caml_hash_state * st, value v)
+{
+  struct caml_ba_array * b = Caml_ba_array_val(v);
+  uintnat num_elts, i;
+  int n;
+  uint64_t w;
+
+  num_elts = 1;
+  for (n = 0; n < b->num_dims; i++) num_elts = num_elts * b->dim[n];
```

Comment:
This is the appveyor failure:
```suggestion
  for (n = 0; n < b->num_dims; n++) num_elts = num_elts * b->dim[n];
```